lua/graph.lua

```diff
@@ -1256,6 +1262,10 @@ function filter_tags_generic(kv)
     else
       kv["pedestrian_backward"] = kv["pedestrian_forward"]
     end
+
+  -- what on earth is going on here? if there is no oneway tagging we say the way is bidirectional despite all the
+  -- backward/forward crap we did above? this has to be the most common case (no oneway tag) so why on earth does it
+  -- reset any of the other directional tagging that is parsed above?
   elseif oneway_norm == nil or oneway_norm == "false" then
```

Comment:
i had tried putting the `:forward` overrides just above this with some of the bike and pedestrian stuff but this bit of code immediately neuters that. i cant really make sense of it, seems like we are assumign we only processed forward so far and no oneway means we can copy it over to backward. that would make sense but just above it there is a bunch of crap that sets `_backward` so.. not sure..